extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.js

```diff
@@ -43,6 +47,12 @@ const LightboxControlsModes = {
 
 const DESC_BOX_PADDING_TOP = 50;
 
+/** @private @const {!../../../src/curve.CurveDef} */
```

Comment:
No need for `@private`  and `_` at the end since they are not exported. will make them consistent with other consts here.